iib/web/models.py

```diff
@@ -911,6 +912,7 @@ def from_json(cls, kwargs, batch=None):
         if not isinstance(request_kwargs.get('force_backport', False), bool):
             raise ValidationError('"force_backport" must be a boolean')
 
+        # Why you check crn_token and force_backport above and remove it here?
```

Comment:
I went through the add bundles but I don't understand why it is used this way @yashvardhannanavati

Reply:
Sensitive information should not be stored in DB directly. 
Not sure about `force_backport`.

Reply:
thanks